# A chromatin-based regulator of RNA Polymerase III assembly at tRNA genes revealed by locus-specific proteomics


### Maria Elize van Breugel <sup>1</sup>, Ila van Kruijsbergen <sup>1</sup>, Chitvan Mittal <sup>2,7</sup>, Cor Lieftink <sup>3</sup>, Ineke Brouwer <sup>1</sup>, Teun van den Brand <sup>1</sup>, Roel Kluin <sup>4</sup>, Ren√©e Menezes <sup>5</sup>, Tibor van Welsem <sup>1</sup>, Andrea Del Cortona <sup>6</sup>, Muddassir Malik <sup>1</sup>, Tineke Lenstra <sup>1</sup>, Elzo de Wit <sup>1</sup>, Kevin Verstrepen <sup>6</sup>, B. Franklin Pugh <sup>7</sup>, Fred van Leeuwen1 <sup>1,8*</sup>


<sup>1</sup>Division of Gene Regulation, Netherlands Cancer Institute, 1066CX Amsterdam, The Netherlands
<sup>2</sup>Baker Institute for Animal Health, College of Veterinary Medicine, Cornell University, Ithaca, NY 14853, USA
<sup>3</sup>High Throughput Screening Facility, Netherlands Cancer Institute, 1066CX Amsterdam, The Netherlands
<sup>4</sup>Genomics Core Facility, Netherlands Cancer Institute, 1066CX Amsterdam, The Netherlands
<sup>5</sup>Biostatistics Unit, Netherlands Cancer Institute, 1066CX Amsterdam, The Netherlands
<sup>6</sup>Center for Microbiology, KU Leuven, 3001 Heverlee-Leuven, Gaston Geenslaan 1, Leuven, Belgium
<sup>7</sup>Department of Molecular Biology and Genetics, Biotechnology Building, Cornell University, Ithaca, NY 14853, USA 
<sup>8</sup>Department of Medical Biology, Amsterdam UMC, University of Amsterdam, Meibergdreef 9, 1105AZ Amsterdam, The Netherlands


### Correspondence: Fred van Leeuwen: fred.v.leeuwen@nki.nl


### PMID : [XXXXXXXX](https://pubmed.ncbi.nlm.nih.gov/XXXXXXXX/)
### GEO ID : [XXXXXXXX](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=XXXXXXXX)

## Abstract
Transfer RNA genes (tDNAs) are involved in a wide variety of cellular processes ranging from mRNA translation to genome organization and disease. Transcription of tDNAs by RNAPIII is tightly regulated upon nutrient and stress signaling. However, the emerging notion that tDNAs across the genome are differentially regulated suggests that chromatin-based regulatory mechanisms are at play. To identify such mechanisms, we used Epi-Decoder to decode the chromatin proteome of a single native tDNA locus in the yeast genome by DNA-sequencing. The tDNA proteome, including the core RNAPIII transcription machinery, showed dynamic reprogramming upon conditions of nutrient stress. Among the factors that dynamically occupy the tDNA proteome we identified Ykr011c (referred to as Fpt1), a protein of unknown function. Fpt1 uniquely binds tRNA genes but occupancy at tRNA genes was variable and correlated with RNAPIII responsiveness. Decoding the tDNA proteome in the absence of Fpt1 revealed a role in the eviction of RNAPIII during conditions of repressed transcription. While Fpt1 exclusively occupied RNAPIII regulated genes, cells without Fpt1 also showed impaired shutdown of RNAPII transcribed ribosome biogenesis genes in changing nutrient conditions. Together, a DNA-sequencing based locus-specific proteome atlas revealed dynamic binding of known and unknown factors in the tDNA proteome. Our findings provide support for a chromatin-based regulatory mechanism of RNAPIII assembly that also tunes an integrated physiological response to changing metabolic conditions.


## Table of Contents

### data

### bin

### 00_Download_and_Preprocessing

### 01_GenoPipe

### 0X_Bulk_Processing
